packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -114,6 +120,17 @@ const CreateAdForm = () => {
                 {({ errors, handleChange, isSubmitting, isValid, setFieldValue, touched, values }) => {
                     const is_sell_advert = values.type === buy_sell.SELL;
 
+                    const onChangeAdTypeHandler = user_input => {
+                        setFieldValue('type', user_input);
+                        if (floating_rate_store.rate_type === ad_type.FLOAT) {
+                            if (user_input === buy_sell.SELL) {
+                                setFieldValue('price_rate', '0.01');
+                            } else {
+                                setFieldValue('price_rate', '-0.01');
```

Comment:
`price_rate` isn't clear, please change it to `rate_type`.

Reply:
Will need to change in a lot of places. Please advice